RecoParticleFlow/Configuration/python/RecoParticleFlow_cff.py

```diff
@@ -86,4 +86,9 @@
     e.toModify(pfPileUp, enable = cms.bool(False))
     
 
+from Configuration.ProcessModifiers.mlpf_cff import mlpf
+from RecoParticleFlow.PFProducer.mlpfProducer_cfi import mlpfProducer
 
+mlpfTask = cms.Task(mlpfProducer, particleFlowRecoTask.copy())
```

Comment:
```suggestion
_mlpfTask = cms.Task(mlpfProducer, particleFlowRecoTask.copy())
```
to avoid exporting this, only `particleFlowRecoTask` needs to be visible

Reply:
fixed in 760dfed4737